ngi_reports/stockholm/project_summary.py

```diff
@@ -77,29 +84,49 @@ def __init__(self, config, LOG, working_dir, **kwargs):
         self.project_info['UPPMAX_path'] = "/proj/{}/INBOX/{}".format(self.project_info['UPPMAX_id'], self.project_info['ngi_name'])
-        self.project_info['ordered_reads'] = self.get_ordered_reads()
+        self.project_info['ordered_reads'] = []
         self.project_info['best_practice'] = False if self.proj_details.get('best_practice_bioinformatics','No') == "No" else True
         self.project_info['status'] = "Sequencing done" if self.proj.get('project_summary', {}).get('all_samples_sequenced') else "Sequencing ongoing"
         self.project_info['library_construction'] = self.get_library_method()
         self.project_info['accredit'] = self.get_accredit_info(['library_preparation','sequencing','data_processing','data_analysis'])
-
-        ## Collect information about the sample preps
+        self.project_info['total_lanes'] = 0
+        self.project_info['display_limit'] = kwargs.get('display_limit')
+        self.project_info['missing_fc'] = False
+        self.project_info['aborted_samples'] = {}
+        
+        ## Collect information about the sample preps and collect aborted samples
         for sample_id, sample in sorted(self.proj.get('samples', {}).iteritems()):
             self.LOG.info('Processing sample {}'.format(sample_id))
+            ## Check if the sample is aborted before processing
+            if sample.get('details',{}).get('status_(manual)') == "Aborted":
+                self.LOG.info('Sample {} is aborted, so skipping it'.format(sample_id))
+                self.project_info['aborted_samples'][sample_id] = {'user_id': sample.get('customer_name',''), 'status':'Aborted'}
+                continue
+            
             ## Basic fields from Project database
             self.samples_info[sample_id] = {'ngi_id': sample_id}
             self.samples_info[sample_id]['customer_name'] = sample.get('customer_name','')
-            self.samples_info[sample_id]['total_reads'] = sample.get('details',{}).get('total_reads_(m)','')
-            if self.project_info['ordered_reads']:
-                self.samples_info[sample_id]['seq_status'] = ['FAIL','PASS'][float(self.samples_info[sample_id]['total_reads']) > \
-                                                                         float(self.project_info['ordered_reads'].replace('M',''))]
-            else:
-                self.samples_info[sample_id]['seq_status'] = None
+            self.samples_info[sample_id]['total_reads'] = sample.get('details',{}).get('total_reads_(m)')
+            self.samples_info[sample_id]['reads_min'] = sample.get('details',{}).get('reads_min')
+            
+            ## Check if a non-aborted sample is sequenced
+            if self.samples_info[sample_id]['total_reads'] == None:
+                self.LOG.warn("Sample {} dont have total reads, so adding it to NOT sequenced samples list.".format(sample_id))
```

Comment:
I not good with `Grammar` :sweat_smile: 

![image](https://cloud.githubusercontent.com/assets/5311007/8478690/43c62212-20d3-11e5-969d-fd2f45fce8ad.png)